CONTRIBUTING.md

```diff
@@ -292,14 +293,30 @@ When you edit a blank section:
 
 - If your PR adds a **breaking change**, create a new heading called `#### BREAKING CHANGES` and add
   bullet point that explains the breaking changes; provide a migration path if possible.
-- If your PR adds a new feature of a fix, create a new heading called `#### Other changes` and
-  add a bullet point that explains the fix or the new feature. Make sure that this new heading
-  appears after the `#### BREAKING CHANGES` heading.
+  Read [how we version Biome](https://biomejs.dev/internals/versioning/) to determine if your change is a breaking change.
+  A breaking change result in a major release.
+- If your PR adds a new feature, enhance an existing feature, or fix a bug, create a new heading called `#### New features`, `#### Enhancements`, or `#### Bug fixes`.
+  Then, add a bullet point that explains the change.
+
+Make sure that the created subsections are ordered in the following order:
+
+```md
+#### BREAKING CHANGES
+
+#### New features
+
+#### Enhancements
```

Comment:
What's the difference between enhancements and features?

Reply:
`enhancements` improve an existing feature (for example extending cases covered by a rule). `New features` includes new added rules, new config options, ...